seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-kafka/src/main/java/org/apache/seatunnel/flink/kafka/source/KafkaTableStream.java

```diff
@@ -110,7 +111,12 @@ public void prepare(FlinkEnvironment env) {
         }
         String schemaContent = config.getString(SCHEMA);
         format = FormatType.from(config.getString(SOURCE_FORMAT).trim().toLowerCase());
-        schemaInfo = JsonUtils.parseArray(schemaContent);
+        try {
+            schemaInfo = JsonUtils.stringToJsonNode(schemaContent);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
I don't have any other questions, it's good enough to be able to throw specific exception information, like this, SchemaConvertException?

Reply:
> I don't have any other questions, it's good enough to be able to throw specific exception information, like this, SchemaConvertException?

yes,I think you're right. I will update the code.